lib/themes/helpers.js

```diff
@@ -84,3 +84,14 @@ export const getTheme = (themeIdentifier) => {
  * @returns {String}
  */
 export const toStyle = (themes = []) => themes.join('\n');
+
+/**
+ * Given two arguments, one meant to be used in dark mode and the other one in the other cases,
+ * pick the appropiate one depending on whether the class 'isDarkMode' is in the body or not
+ * @param dark
+ * @param light
+ * @return {*}
+ */
+export const getLightOrDark = ({ dark, light }) => {
+    return document.body.classList.contains('isDarkMode') ? dark : light;
```

Comment:
Probably missed to use `DARK_MODE_CLASS` here

Reply:
Likelihood = 100%